Assignment6.Rmd

```diff
@@ -77,7 +76,7 @@ table(M2$certified, M2$predict2)
 Choose a data file from the (University of Michigan Open Data Set)[https://github.com/bkoester/PLA/tree/master/data]. Choose an outcome variable that you would like to predict. Build two models that predict that outcome from the other variables. The first model should use raw variables, the second should feature select or feature extract variables from the data. Which model is better according to the cross validation metrics?
 
 ```{r}
-
+A1 <- read.csv("https://github.com/bkoester/PLA/blob/master/data/student.course.csv")
```

Comment:
Keep trying part 3 to practice model comparison for classification trees! Looking forward to seeing that.